integration-tests/features/steps/three_scale_preview.py

```diff
@@ -51,7 +52,8 @@ def send_manifest_to_stack_analyses(context, manifest, name, endpoint, user_key,
     if user_key:
         for _ in range(rate):
```

Comment:
I dont think we should fire these many requests

Reply:
this is a common function with is also used during load test so that is why it has a loop but any how it will make only one request